draft-ietf-scitt-architecture.md

```diff
@@ -517,29 +521,27 @@ All Transparency Services MUST expose standard endpoints for Registration of Sig
 
 ### Service Identity, Remote Attestation, and Keying
 
-Every Transparency Service MUST have a public service identity,
-associated with public/private key pairs for signing on behalf of the service.
+Every Transparency Service MUST have a public service identity, associated with public/private key pairs for signing on behalf of the service.
 In particular, this identity must be known by Verifiers when validating a Receipt.
 
-This identity should be stable for the lifetime of the service, so that all Receipts remain valid and consistent.
+This identity MUST be stable for the lifetime of the service, so that all Receipts remain valid and consistent. 
 The Transparency Service operator MAY use a distributed identifier as their public service identity if they wish to rotate their keys, if the Registry algorithm they use for their Receipt supports it.
 Other types of cryptographic identities, such as parameters for non-interactive zero-knowledge proof systems, may also be used in the future.
 
-A Transparency Services SHOULD provide evidence that it is securely implemented and operated, enabling remote authentication of the hardware platforms and/or software TCB that run the Transparency Service.
-This additional evidence SHOULD be recorded in the Registry and presented on demand to Verifiers and Auditors.
+A Transparency Service MAY provide extra evidence that it is securely implemented and operated, enabling remote authentication of the hardware platforms and/or software TCB that run the Transparency Service.
+If present, this additional evidence MUST be recorded in the Registry and presented on demand to Verifiers and Auditors.
 Examples for Statements that can improve trustworthy assessments of Transparency Services are RATS Conceptual Messages, such as Evidence, Endorsements, or corresponding Attestation Results (see {{-rats-arch}}.
 
 For example, consider a Transparency Service implemented using a set of replicas, each running within its own hardware-protected trusted execution environments (TEEs).
-Each replica SHOULD provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
-This attestation evidence SHOULD be supplemented with transparency Receipts for the software and configuration of the service, as measured in its attestation report.
+Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
+This attestation evidence can be supplemented with transparency Receipts for the software and configuration of the service, as measured in its attestation report.
```

Comment:
```suggestion
This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
```

Reply:
I don't want to have normative language on attestation in this document because so far attestation hasn't been integrated. It appears strange to suddenly bring it up here.

Reply:
+1